adminLogin.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Login Form</title>
+</head>
+
+<body>
+<?php
+session_start();
+if ($_SESSION['userLoggedIn']){
+    header("Location: cmsHomePage.php");
```

Comment:
does this work? i dont think it will, u cant set header locations after html output